StorageTests/TestSQLiteHistory.swift

```diff
@@ -662,10 +662,10 @@ class BrowserSchemaV10: BaseHistoricalBrowserSchema {
 
         let args: Args = [
             root, BookmarkRoots.RootGUID, type, "Root", root,
-            BookmarkRoots.MobileID, BookmarkRoots.MobileFolderGUID, type, titleMobile, root,
-            BookmarkRoots.MenuID, BookmarkRoots.MenuFolderGUID, type, titleMenu, root,
-            BookmarkRoots.ToolbarID, BookmarkRoots.ToolbarFolderGUID, type, titleToolbar, root,
-            BookmarkRoots.UnfiledID, BookmarkRoots.UnfiledFolderGUID, type, titleUnsorted, root,
+            1, BookmarkRoots.MobileFolderGUID, type, titleMobile, root,
+            2, BookmarkRoots.MenuFolderGUID, type, titleMenu, root,
+            3, BookmarkRoots.ToolbarFolderGUID, type, titleToolbar, root,
+            4, BookmarkRoots.UnfiledFolderGUID, type, titleUnsorted, root,
```

Comment:
nit: this code is repeated a lot! Is existing/old code though, not sure this is the time to fix it.